include/oneapi/dpl/pstl/hetero/algorithm_ranges_impl_hetero.h

```diff
@@ -464,9 +599,9 @@ __pattern_unique(__hetero_tag<_BackendTag> __tag, _ExecutionPolicy&& __exec, _Ra
     auto res = __ranges::__pattern_unique_copy(__tag, __exec, __rng, res_rng, __pred,
                                                oneapi::dpl::__internal::__pstl_assign());
 
-    __ranges::__pattern_walk_n(__tag, ::std::forward<_ExecutionPolicy>(__exec),
-                               __brick_copy<__hetero_tag<_BackendTag>, _ExecutionPolicy>{}, res_rng,
-                               ::std::forward<_Range>(__rng));
+    __ranges::__pattern_walk_n(__tag, std::forward<_ExecutionPolicy>(__exec),
+                     __brick_copy<__hetero_tag<_BackendTag>, _ExecutionPolicy>{}, res_rng,
+                     std::forward<_Range>(__rng));
```

Comment:
same as above:  
I believe this change is "formatting only", and also may have a merge conflict with changes merged today from #1765 .

Reply:
It seems Mikhail reverted it. This is not relevant anymore.